### PR TITLE
DVAD-3154: Fix vulnerabilities

### PR DESCRIPTION
**Pull request by haggishunt56**
This PR fixes the security vulnerabilities in these tickets:
https://collaboration.homeoffice.gov.uk/jira/browse/DVAD-3150
https://collaboration.homeoffice.gov.uk/jira/browse/DVAD-3154

### CODE REVIEW

**Comment by JamesWoodwardHMPO** [comment]
No changes to package.json?

File: .nvmrc [hunk 1]
```diff
@@ -0,0 +1 @@
+v12
```
Comment (aarongill1) [inline .nvmrc:1]: Do we need this file committed? No harm I guess

**Review by aarongill1 — APPROVED** [review]
We should start to add the patch version in the package.json to start tracking changes - I know we've not been doing this recently but...

  "name": "lev-api",
  "version": "0.16.5",

File: .nvmrc (hunk 1 above)
Comment (JamesWoodwardHMPO) [reply .nvmrc:1]: I think it's useful to have it committed, gives you a hint when nothing compiles that you're on the wrong version of NodeJS.

**Review by aarongill1 — APPROVED** [review]
LGTM

**Pull request merged**